docs/INSTALLATION.rst

```diff
@@ -1,93 +1,86 @@
 Installation
 ============
 
-*pip* is the recommended installer for installing and managing Python packages.
-The project can be installed either locally or from the Python Package Index.
-All `TUF releases
-<https://github.com/theupdateframework/python-tuf/releases>`_ are cryptographically
-signed, with GPG signatures available on both GitHub and `PyPI
-<https://pypi.python.org/pypi/tuf/>`_.  PGP key information for our maintainers
-is available on our `website
-<https://theupdateframework.github.io/people.html>`_, on major keyservers,
-and on the `maintainers page
-<https://github.com/theupdateframework/python-tuf/blob/develop/docs/MAINTAINERS.txt>`_.
+All versions of ``python-tuf`` can be installed from
+`PyPI <https://pypi.org/project/tuf/>`_ with
+`pip <https://pip.pypa.io/en/stable/>`_.
 
+::
 
-Release Verification
---------------------
+   python3 -m pip install tuf
 
-Assuming you trust `the maintainer's PGP key
-<https://github.com/theupdateframework/python-tuf/blob/develop/docs/MAINTAINERS.txt>`_,
-the detached ASC signature can be downloaded and verified.  For example::
+By default tuf is installed as pure python package with limited cryptographic
+abilities. See `Install with full cryptographic abilities`_ for more options.
 
-   $ gpg --verify securesystemslib-0.10.8.tar.gz.asc
-   gpg: assuming signed data in 'securesystemslib-0.10.8.tar.gz'
-   gpg: Signature made Wed Nov  8 15:21:47 2017 EST
-   gpg:                using RSA key 3E87BB339378BC7B3DD0E5B25DEE9B97B0E2289A
-   gpg: Good signature from "Vladimir Diaz (Vlad) <vladimir.v.diaz@gmail.com>" [ultimate]
 
+Verify release signatures
+-------------------------
```

Comment:
I find this section sticking out a bit too much for installation instructions. Should we shorten it, or at least move it to the bottom of the document?

Reply:
I think I agree. I'd like to promote signature verification but when the process looks like this (even while handwaving the PGP key lookup part)... I think it breaks the flow of the document and hides the other install options below the fold.

Considering we have three different install methods (normal, native crypto, dev)... maybe list those three first and then verification as the last thing?

Reply:
see 0216740